src/Umbraco.Web/Search/UmbracoTreeSearcher.cs

```diff
@@ -28,12 +29,37 @@ internal class UmbracoTreeSearcher
         /// <param name="pageSize"></param>
         /// <param name="pageIndex"></param>
         /// <returns></returns>
+        [Obsolete("This method is obsolete, use the overload with ignoreUserStartNodes instead", false)]
```

Comment:
why not just add `ignoreUserStartNodes = false` to as another param? If we are worried about people having to rebuild their projects, then we will need to change a lot more in this PR! otherwise no need for an extra overload.